Classes/Controller/PageViewController.php

```diff
@@ -111,7 +111,7 @@ protected function getFulltext($page)
             $physicalStructureInfo = $this->document->getCurrentDocument()->physicalStructureInfo[$this->document->getCurrentDocument()->physicalStructure[$page]];
             $fileId = $physicalStructureInfo['files'][$fileGrpFulltext];
             if (!empty($fileId)) {
-                $file = $this->document->getCurrentDocument()->getFileLocation($fileId);
+                $file = $this->document->getCurrentDocument()->getFileInfo($fileId);
```

Comment:
Is this change intended or some kind of merging error?

Reply:
It is necessary, in some point I have made error while solving merging conflicta which got already to master and now it reverses it to the correct state.